stable/magic-namespace/values.yaml

```diff
@@ -23,6 +23,28 @@ tiller:
 
   maxHistory: 0
 
+  tls:
+    ## Enable TLS encryption between Helm and Tiller
+    enabled: false
+
+    ## Verify remote certificate
+    verify: true
+
+    ## The directory where Tiller's TLS certificates are located
+    certPath: /etc/certs
```

Comment:
Any particular reason this is configurable? It seems to me that you could hardcode this as the value of the `TILLER_TLS_CERTS` environment variable and as the the `mountPath` for the volume and that would always work. I'm not sure what value the user could get out of overriding that.

Reply:
I thought perhaps that some organisations may be enforcing a path to mount to i.e. `/ssl` for all containers but it's probably pretty far fetched so I'll make changes to hardcode them

Reply:
@krancour ok that's hardcoded now.